Simplified/it.lproj/Localizable.strings

```diff
@@ -31,6 +31,7 @@
 "DownloadCouldNotBeCompleted" = "Il download è fallito.";
 "DownloadCouldNotBeCompletedFormat" = "Il download di %@ non può essere completato.";
 "The download could not be completed.\nScroll down to 'View Issues' to see details." = "Il download non è stato completato.\nScorri in fondo alla pagina per i dettagli dell'errore.";
+"The download could not be completed." = "Il download non è stato completato.";
```

Comment:
perfect translation!

Reply:
I copied and pasted from the line above haha